app/Models/Comment.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model as Eloquent;
+
+class Comment extends Eloquent
+{
+	 public function news()
+     {
+        return $this->hasOne('App\Models\News');
```

Comment:
untuk relasi **one-to-many** silahkan lihat di https://laravel.com/docs/5.3/eloquent-relationships#one-to-many , pada hal ini bisa menggunakan `belongsTo` sebagai inverse dari hasMany